src/compute-client/src/explain.rs

```diff
@@ -27,14 +27,14 @@
 //! printed in contexts where trailing whitespace is unacceptable, like
 //! sqllogictest files.
 
+#![deprecated = "Use `explain_new` instead."]
+
 use std::fmt;
 
```

Comment:
The `pub trait ViewFormatter` in this file has no implementations according to my IDE. Maybe you intended to delete this file?

You also wrote
> Remove two of the three explain.rs files

but there seem to be 2 `explain.rs` files remaining.

Reply:
I forgot to remove the one in `compute-client`.